cms-common/src/main/java/io/commercetools/sunrise/cms/CmsServiceException.java

```diff
@@ -0,0 +1,8 @@
+package io.commercetools.sunrise.cms;
+
+public class CmsServiceException extends RuntimeException {
```

Comment:
Can you add some documentation here about the exception? You can find some inspiration in this one:
https://github.com/commercetools/commercetools-sunrise-java-email/blob/master/email-common/src/main/java/io/commercetools/sunrise/email/EmailDeliveryException.java